state/core_access.go

```diff
@@ -594,7 +594,7 @@ func (ca *CoreAccessor) setupTxClient(ctx context.Context, keyName string) (*use
 	}
 	ca.defaultSignerAddress = addr
 	return user.SetupTxClient(ctx, ca.keyring, ca.coreConn, encCfg,
-		user.WithDefaultAccount(keyName), user.WithDefaultAddress(addr),
```

Comment:
why removing this in this PR? (and why in general?)

Reply:
It was a temporary solution, as I my node crashed multiple times(in the app code) on the previous version of app. Will revert changes since we've bumped it

Reply:
let's revert it yea